source/actions3/actions/AMControlMoveAction3.cpp

```diff
@@ -85,15 +114,21 @@ void AMControlMoveAction3::startImplementation()
 	// remember the start position:
 	startPosition_ = control_->toInfo();
 
-	// start the move:
-	int failureExplanation;
+	attempts_ = 1;
+
+	// determine the absolute destination
 	if(controlMoveInfo()->isRelativeMove() && controlMoveInfo()->isRelativeFromSetpoint())
-		failureExplanation = control_->moveRelative(setpoint.value(), AMControl::RelativeMoveFromSetpoint);
+		destination_ = control_->setpoint() + setpoint.value();
```

Comment:
Nope, this is what we tested yesterday. Seemed to work correctly as far as we could tell.